app/controllers/subscription_controller.rb

```diff
@@ -119,19 +119,21 @@ def digest
   end
 
   def multiple_add
-    unless params[:names]
+    #params[:same] contains the list of tag names separated by comma
+    unless params[:same]
```

Comment:
Why are you using :same?
Whenever we create any segment of code we must think about those contributors who will be there when we will be not present here. Here comes the role of readability. same is ambiguous keyword. Names is a better keyword. Please change this back to names.

Reply:
Yes sir that's what I started with initially, but that wasn't working, probably because of some keyword or some other similar name, that's why I was forced to change it. I truly understand that readability is very important. I am so sorry if this was too big a problem.

Reply:
I have changed bare minimum code for controller and routes because I understand that a lot of thought has been put behind it to make it the best.

Reply:
Let's change it to tagnames that will be much better if names is creating some problem.

Reply:
Here you go. I am sorry again.